controllers/error-handling-controllers.js

```diff
@@ -1,3 +1,19 @@
+exports.nonExistentPath404 = (req, res, next) => {
+  res.status(404).send({ msg: "Path Not Found" });
+};
+
+exports.errorHandler400 = (err, req, res, next) => {
+  if (err === "Invalid ID provided") {
+    res.status(400).send({ msg: err });
+  } else next(err);
+};
+
+exports.errorHandler404 = (err, req, res, next) => {
+  if (err === "ID not found") {
+    res.status(404).send({ msg: err });
+  } else next(err);
+};
```

Comment:
this is like half of a custom error handler and half of a 404. maybe you could make it more reusable?